python/invoke_release/tasks.py

```diff
@@ -1198,13 +1259,19 @@ def release(_, verbose=False, no_stash=False):
 
         _pre_commit(__version__, release_version)
 
+        if USE_PULL_REQUEST:
+            branch_name = 'invoke-release-{}'.format(release_version)
+            _create_branch(verbose, branch_name)
         _commit_release_changes(release_version, cl_message, verbose)
 
         _pre_push(__version__, release_version)
 
-        _tag_branch(release_version, cl_message, verbose)
+        if USE_TAG:
+            _tag_branch(release_version, cl_message, verbose)
         pushed_or_rolled_back = _push_release_changes(release_version, branch_name, verbose)
 
+        if USE_PULL_REQUEST:
+            _checkout_branch(verbose, BRANCH_MASTER)
         _post_release(__version__, release_version, pushed_or_rolled_back)
 
         _standard_output('Release process is complete.')
```

Comment:
If the project is using pull requests, I believe this should say something different:

>You're almost done! The release process will be complete when you create a pull request and it is merged.